amv/.htaccess

```diff
@@ -0,0 +1,74 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+
+RewriteEngine on
+
+# Rewrite rule for latest version.
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://isibang.ac.in/ns/amv/index.html [R=303,L]
```

Comment:
The letsencrypt cert for this site is for `www.isibang.ac.in` only.  I'd adjust the setup for that to ensure it has `isibang.ac.in` as a subject alt name as well.  Otherwise browsers and tools will likely have connection issues.

Reply:
Yes, actually both work. But best is to use www.isibang.ac.in.  I will update .htaccess file. Is this okay?

Reply:
You can use whatever you'd like.  I was just pointing out the certificate problems with the current setup.